src/state_machine.zig

```diff
@@ -663,6 +667,7 @@ pub fn StateMachineType(
 
         pub fn prefetch(
             self: *StateMachine,
+            client_release: vsr.Release,
             callback: *const fn (*StateMachine) void,
```

Comment:
Imo `callback` should be the first argument (by convention), then `client_release`.